06_confidence_intervals/confidence_intervals.Rmd

```diff
@@ -85,46 +95,52 @@ variable `area`.
 ## Confidence intervals
 
 Return for a moment to the question that first motivated this lab: based on 
-this sample, what can we infer about the population? Based only on this single 
-sample, the best estimate of the average living area of houses sold in Ames 
-would be the sample mean, usually denoted as $\bar{x}$ (here we're calling it 
-`x_bar`). That serves as a good **point estimate** but it would be useful 
-to also communicate how uncertain we are of that estimate. This uncertainty
-can be quantified using a **confidence interval**.
+this sample, what can you infer about the population? With just one sample, the 
+best estimate of the average living area of houses sold in Ames would be the sample
+mean, usually denoted as $\bar{x}$ (here we're calling it `x_bar`). That serves 
+as a good **point estimate**, but it would be useful to also communicate how uncertain
+you are of that estimate. This uncertainty can be quantified using a **confidence
+interval**.
 
 A confidence interval for a population mean is of the following form
 \[ \bar{x} + z^\star \frac{s}{\sqrt{n}} \]
 
-You should by now be comfortable with calculating the mean and standard deviation of 
-a sample in R. And we know that the sample size is 60. So the only remaining building
-block is finding the appropriate critical value for a given confidence level. We can
-use the `qnorm` function for this task, which will give the critical value associated
-with a given percentile under the normal distribution. Remember that confidence levels
-and percentiles are not equivalent. For example, a 95% confidence level refers to the
-middle 95% of the distribution, and the critical value associated with this area will
-correspond to the 97.5th percentile.
-
-We can find the critical value for a 95% confidence interal using
-```{r z_star_95}
-z_star_95 <- qnorm(0.975)
-z_star_95
-```
-which is roughly equal to the value critical value 1.96 that you're likely
-familiar with by now.
+Although you can do this calculate by hand, the `infer` package provides a set of
+commands consistent with the `tidyverse` framework, thus making it easy to follow 
+a similar workflow. You can use these commands to find a single confidence 
+interval:
+
+Command | Purpose
+------- | -------
+`specify` | Identify your variable of interest using the `formula` argument
+`generate` | The amount of repetitions you want to do
+`calculate` | The type of statistic you are looking for
```

Comment:
"The type of statistic you are looking for" = "The sample statistic you want to do inference with, or you can also think of this as the population parameter you want to do inference for"